playwright.config.js

```diff
@@ -34,44 +35,37 @@ module.exports = defineConfig({
   },
 
   /* Configure projects for major browsers */
-  projects: [
-    { name: 'setup', testMatch: /.*\.setup\.js/ },
-    {
-      name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
-      dependencies: ['setup']
-    },
+  projects: testConfig.projects,
 
-    // {
-    //   name: 'firefox',
-    //   use: { ...devices['Desktop Firefox'] },
-    // },
+  // {
```

Comment:
See above point on multiple browsers, thinking now it should probably be in a different ticket so I'll write it up now but these comments can probably just be removed.

Reply:
https://dsdmoj.atlassian.net/browse/CRM457-2329